cibuildwheel/ios.py

```diff
@@ -235,6 +257,7 @@ def setup_python(
     dependency_constraint_flags: Sequence[PathOrStr],
     environment: ParsedEnvironment,
     build_frontend: BuildFrontendName,
+    safe_tools: Sequence[str],
```

Comment:
the same trick that was used on `cross_virtualenv` to silence the `Too many positional arguments (6/5)` can be used here (i.e., add `*,` & update the call to use kwargs)